core/src/test/scala/org/broadinstitute/dsde/rawls/monitor/workspace/WorkspaceResourceMonitorSpec.scala

```diff
@@ -0,0 +1,65 @@
+package org.broadinstitute.dsde.rawls.monitor.workspace
+
+import org.broadinstitute.dsde.rawls.monitor.WorkspaceResourceMonitor
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatestplus.mockito.MockitoSugar.mock
+
+class WorkspaceResourceMonitorSpec extends AnyFlatSpec {
+
+  import org.mockito.MockedStatic
+  import org.mockito.Mockito
+
+  behavior of "static methods"
+
+  it should "work" in {
+    val m = mock[WorkspaceResourceMonitor]
+    m.companion.pureFunction()
+//WorkspaceResourceMonitor.p
+    try {
+
+      val monitorCompanion = Mockito.mockStatic(classOf[WorkspaceResourceMonitor]) //WorkspaceResourceMonitor.getClass.c)
+     //monitorCompanion.p
+      //monitorCompanion.com
+    }
+  }
+
+  /*
+  @Test  def lookMomICanMockStaticMethods(): Unit =  { assertThat(Buddy.name).isEqualTo("John")
```

Comment:
Hah, I appreciate the help, but no need to merge this in. :)

Reply:
woops